src/wallet/rpc/transactions.cpp

```diff
@@ -322,6 +322,16 @@ static void MaybePushAddress(UniValue & entry, const CTxDestination &dest)
     }
 }
 
+//! Fetch wallet descriptors matching this scriptPubKey.
+static void MaybePushWalletDescriptors(const CWallet& wallet, const CScript& script_pubkey, UniValue& entry)
```

Comment:
In 8976cd89051d262bfed2498de5aa9ed190e3b331 "rpc: output wallet descriptors for received entries in listsinceblock"

nit: There's no maybe here, it will always push `wallet_descs`.